# Publish joint states (positions/velocities/efforts)
joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

my_balance_controller:
  type: "simble_box_bot/BalanceController" 
  left_wheel_name: "left_wheel_joint"
  right_wheel_name: "right_wheel_joint"
  kp: 12.0
  ki: 0
  kd: 0.8

  wheel_radius: 0.06
  track_width: 0.175

  vel_kp: 0.2
  vel_ki: 0.05

  yaw_kp: 1.0 
  yaw_ki: 0.0   
  yaw_kd: 0.05